modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v550/RegistryDataManager.java

```diff
@@ -215,4 +336,28 @@ private void updateRegistryProperties(Registry registry, String resource, List<S
             }
         }
     }
+
+    private List<String> getSTSPolicyPaths(Registry registry) throws RegistryException {
+
+        List<String> policyPaths = new ArrayList<>();
+        if (registry.resourceExists(SERVICE_GROUPS_PATH)) {
+            Collection serviceGroups = (Collection)registry.get(SERVICE_GROUPS_PATH);
+            if (serviceGroups != null) {
+                for (String serviceGroupPath : serviceGroups.getChildren()) {
+                    if ( StringUtils.isNotEmpty(serviceGroupPath) &&
+                            serviceGroupPath.contains(STS_SERVICE_GROUP)) {
+                        String policyCollectionPath = new StringBuilder().append(serviceGroupPath)
+                                .append(SECURITY_POLICY_RESOURCE_PATH).toString();
+                        Collection policies = (Collection) registry.get(policyCollectionPath);
+                        if (policies != null) {
+                            for (String policyPath : policies.getChildren()) {
+                                policyPaths.add(policyPath);
```

Comment:
Can't we use policyPaths.addAll(Arrays.asList(policies.getChildren())) instead of the iteration.